### DISCUSSION TITLE
github-repo-size github pages update

### DISCUSSION THREAD

**Discussion by mAmineChniti** [AminoffZ/github-repo-size#51] [General]
I think we should update the github page for this project to reflect the current documentation that we have in the README.md.

**Comment by AminoffZ** [comment]
Done

**Reply by mAmineChniti** [reply]
can you tell me more about this process and i'd love to just automate it with a workflow (as i always do lmao)

**Reply by AminoffZ** [reply]
Look into [github pages](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site). page/ is just an Astro app that creates a static site which is deployed using gh-pages. There is however a typedoc generator included which is pretty neat and generates stuff like https://aminoffz.github.io/github-repo-size/docs/functions/internal_api.getRepoInfo.html There seems to be integrations for workflows.

**Reply by mAmineChniti** [reply]
so basically if i just wrote a workflow that enters the page/ directory and runs the build script it should just simply deploy on its own without needing a key or any of type of secret/env variable ?

**Reply by AminoffZ** [reply]
```
cd page && bun run deploy
```
Should do it ...maybe...
